File: checkSqlite.py
import sqlite3
try:
    conn = sqlite3.connect('prisma/dev.db')
    cursor = conn.cursor()
    cursor.execute('SELECT COUNT(*) FROM User')
    users = cursor.fetchone()[0]
    cursor.execute('SELECT COUNT(*) FROM "Order"')
    orders = cursor.fetchone()[0]
    print(f"SQLite Users: {users}")
    print(f"SQLite Orders: {orders}")
except Exception as e:
    print("Error:", e)

File: checkSqlite.js
const fs = require('fs');
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database('./prisma/dev.db', (err) => {
  if (err) {
    console.error(err.message);
    return;
  }
  db.get('SELECT COUNT(*) as count FROM User', (err, row) => {
    if (err) console.error(err.message);
    else console.log('SQLite Users:', row.count);
  });
  db.get('SELECT COUNT(*) as count FROM "Order"', (err, row) => {
    if (err) console.error(err.message);
    else console.log('SQLite Orders:', row.count);
  });
});
